django/chat/utils.py

```diff
@@ -293,6 +293,7 @@ def sse_string(
             title_llm = OttoLLM()
             await sync_to_async(title_chat)(chat.id, force_title=False, llm=title_llm)
             await sync_to_async(title_llm.create_costs)()
+            await sync_to_async(message.chat.refresh_from_db)()
```

Comment:
This solved a problem with chat titles not updating. (They should update in the sidebar after a message or two)